### PR TITLE
Number Insight API Vonage rebranding

### PR DESCRIPTION
**Pull request by marklewin**
# Description

Change Nexmo references to Vonage

[DEVX-3635](https://nexmoinc.atlassian.net/browse/DEVX-3635)

# Checklist

- [ X] version number incremented (in the `info` section of the spec)

### CODE REVIEW

File: definitions/number-insight.yml [hunk 1]
```diff
@@ -122,7 +122,7 @@ paths:
       description: |
         Provides [advanced number insight](/number-insight/overview#basic-standard-and-advanced-apis) information about a number synchronously, in the same way that the basic and standard endpoints do.
 
-        Nexmo recommends accessing the Advanced API **asynchronously** using the `/advanced/async` endpoint, to avoid timeouts.
+         recommends accessing the Advanced API **asynchronously** using the `/advanced/async` endpoint, to avoid timeouts.
```
Comment (tbedford) [inline definitions/number-insight.yml:125]: Should there be a Vonage here before 'recommends'?

File: definitions/number-insight.yml [hunk 2]
```diff
@@ -1071,12 +1071,12 @@ components:
...
         -- | --
-        0 | Success - request accepted for delivery by Nexmo.
-        1 | Busy - you have made more requests in the last second than are permitted by your Nexmo account. Please retry.
+        0 | Success - request accepted for delivery by .
+        1 | Busy - you have made more requests in the last second than are permitted by your  account. Please retry.
         3 | Invalid - your request is incomplete and missing some mandatory parameters.
         4 | Invalid credentials - the _api_key_ or _api_secret_ you supplied is either not valid or has been disabled.
         5 | Internal Error - the format of the recipient address is not valid.
-        9 | Partner quota exceeded - your Nexmo account does not have sufficient credit to process this request.
+        9 | Partner quota exceeded - your  account does not have sufficient credit to process this request.
```
Comment (tbedford) [inline definitions/number-insight.yml:1079]: Extra space by mistake between 'your' and 'account', or should there be a 'Vonage API' here?

File: definitions/number-insight.yml [hunk 3]
```diff
@@ -1096,12 +1096,12 @@ components:
...
         -- | --
-        0 | Success - request accepted for delivery by Nexmo.
-        1 | Busy - you have made more requests in the last second than are permitted by your Nexmo account. Please retry.
+        0 | Success - request accepted for delivery by .
+        1 | Busy - you have made more requests in the last second than are permitted by your account. Please retry.
         3 | Invalid - your request is incomplete and missing some mandatory parameters.
         4 | Invalid credentials - the _api_key_ or _api_secret_ you supplied is either not valid or has been disabled.
         5 | Internal Error - the format of the recipient address is not valid.
-        9 | Partner quota exceeded - your Nexmo account does not have sufficient credit to process this request.
+        9 | Partner quota exceeded - your  account does not have sufficient credit to process this request.
```
Comment (tbedford) [inline definitions/number-insight.yml:1104]: Same as previous comment.

**Review by tbedford — CHANGES_REQUESTED** [review]

File: definitions/number-insight.yml (hunk 3 above)
Comment (marklewin) [reply definitions/number-insight.yml]: As above

File: definitions/number-insight.yml (hunk 2 above)
Comment (marklewin) [reply definitions/number-insight.yml]: Well spotted!

File: definitions/number-insight.yml (hunk 1 above)
Comment (marklewin) [reply definitions/number-insight.yml]: Yes!

**Review by tbedford — APPROVED** [review]

**Pull request merged**